content/en/docs/reference/command-line-tools-reference/feature-gates/windows-host-containers.md

```diff
@@ -0,0 +1,5 @@
+---
+title: WindowsHostProcessContainers
+content_type: feature_gate
```

Comment:
I apologize if I missed seeing something since this is a big PR, but what happens if we do not specify a `_build`?

Reply:
A small page appears for the feature gate, when we don't want it to.